src/main/java/duke/command/MarkCommand.java

```diff
@@ -0,0 +1,45 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.util.Tasklist;
+
+/**
+ * CS2103T Individual Project AY 21/22 Sem 1
+ * Project Duke
+ *
+ *
+ * Description:
+ * Class that encapsulates the mark task command inputted by the user
+ *
+ * @author Keith Tan
+ */
+public class MarkCommand extends Command {
+
+    public static final String COMMAND_WORD = "add";
+    private final int toMark;
```

Comment:
Perhaps the final variable count be all uppercase? I have noticed this in some other areas as well.

Reply:
I think you may forget to change the final variable to all uppercase.

Reply:
Thanks for raising this up, I believe I had missed out on this. I will rectify it in a future commit.